schema/examples/valid/full.toml

```diff
@@ -6,9 +6,9 @@ version = "0.1.0"
 description = "A project"
 authors = ["Author <author@bla.com>"]
 channels = ["stable"]
-platforms = ["linux-64", "win-64", "osx-64"]
+platforms = ["linux-64", "win-64", "osx-64", "osx-arm64"]
```

Comment:
That file appeared to actually be invalid.

Reply:
Looks like `license_file` should be `license-file`. but in that case, the schema check fails

Reply:
Pushed a fix and added a proper test on the rust side. This was a huge oversight! Thanks for pointing it out!